packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -73,10 +73,10 @@ module.exports = {
         'tests/**', // also common npm pattern
         'spec/**', // mocha, rspec-like pattern
         '**/__tests__/**', // jest pattern
-        'test.js', // repos with a single test file
-        'test-*.js', // repos with multiple top-level test files
-        '**/*.test.js', // tests where the extension denotes that it is a test
-        '**/*.spec.js', // tests where the extension denotes that it is a test
+        'test.{js,jsx}', // repos with a single test file
+        'test-*.{js,jsx}', // repos with multiple top-level test files
+        '**/*.test.{js,jsx}', // tests where the extension denotes that it is a test
+        '**/*.spec.{js,jsx}', // tests where the extension denotes that it is a test
```

Comment:
Just checked to make sure it would work with `minimatch`. It seems like we should be good. I'll push an update.